{"uid":"d01668d44578cbf4","name":"Rename an existing workspace test","fullName":"tests.workspaces_test.TestWorkspaces#test_rename_workspace","historyId":"cd63adf5d11191c5904362b968c9e131","time":{"start":1752285730701,"stop":1752285730701,"duration":0},"description":"Rename an existing workspace","descriptionHtml":"<p>Rename an existing workspace</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"data","time":{"start":1752285728324,"stop":1752285728325,"duration":1},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"_verify_url","time":{"start":1752285728324,"stop":1752285728324,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"session_request","time":{"start":1752285728324,"stop":1752285728324,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1752285728324,"stop":1752285728324,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup_method_fixture","time":{"start":1752285748393,"stop":1752285758825,"duration":10432},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9e757753a <unknown>\n#1 0x55d9e7072f00 <unknown>\n#2 0x55d9e70c2c0c <unknown>\n#3 0x55d9e70c2e31 <unknown>\n#4 0x55d9e7108bd4 <unknown>\n#5 0x55d9e70e75cd <unknown>\n#6 0x55d9e7105f84 <unknown>\n#7 0x55d9e70e7343 <unknown>\n#8 0x55d9e70b478a <unknown>\n#9 0x55d9e70b59de <unknown>\n#10 0x55d9e75412cb <unknown>\n#11 0x55d9e7545242 <unknown>\n#12 0x55d9e752e7ac <unknown>\n#13 0x55d9e7545df7 <unknown>\n#14 0x55d9e7512b2f <unknown>\n#15 0x55d9e75661a8 <unknown>\n#16 0x55d9e7566370 <unknown>\n#17 0x55d9e75763b6 <unknown>\n#18 0x7f981329caa4 <unknown>\n#19 0x7f9813329c3c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/workspaces_test.py\", line 25, in setup_method_fixture\n    login(self.about_page, self.login_page)\n    ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/workspaces_test.py\", line 15, in login\n    about_page.click_login_link()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/pages/about_page.py\", line 18, in click_login_link\n    self.click(self.LOGIN_LINK)\n    ~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/pages/base_page.py\", line 59, in click\n    el: WebElement = self.wait.until(expected_conditions.element_to_be_clickable(locator))\n                     ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Click Login link","time":{"start":1752285748394,"stop":1752285758824,"duration":10430},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9e757753a <unknown>\n#1 0x55d9e7072f00 <unknown>\n#2 0x55d9e70c2c0c <unknown>\n#3 0x55d9e70c2e31 <unknown>\n#4 0x55d9e7108bd4 <unknown>\n#5 0x55d9e70e75cd <unknown>\n#6 0x55d9e7105f84 <unknown>\n#7 0x55d9e70e7343 <unknown>\n#8 0x55d9e70b478a <unknown>\n#9 0x55d9e70b59de <unknown>\n#10 0x55d9e75412cb <unknown>\n#11 0x55d9e7545242 <unknown>\n#12 0x55d9e752e7ac <unknown>\n#13 0x55d9e7545df7 <unknown>\n#14 0x55d9e7512b2f <unknown>\n#15 0x55d9e75661a8 <unknown>\n#16 0x55d9e7566370 <unknown>\n#17 0x55d9e75763b6 <unknown>\n#18 0x7f981329caa4 <unknown>\n#19 0x7f9813329c3c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/pages/about_page.py\", line 18, in click_login_link\n    self.click(self.LOGIN_LINK)\n    ~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/pages/base_page.py\", line 59, in click\n    el: WebElement = self.wait.until(expected_conditions.element_to_be_clickable(locator))\n                     ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Rename an existing workspace","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","steps":[{"name":"public ip address","time":{"start":1752285733231,"stop":1752285733242,"duration":11},"status":"passed","steps":[],"attachments":[{"uid":"b92c1379081cfb2e","name":"HTTP logs of http://checkip.amazonaws.com/","source":"b92c1379081cfb2e.txt","type":"text/plain","size":357}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"48795dc678aa9e1e","name":"Cookies","source":"48795dc678aa9e1e.json","type":"application/json","size":4114},{"uid":"68548a19d6114874","name":"Session Storage","source":"68548a19d6114874.json","type":"application/json","size":358},{"uid":"277355b5a1564354","name":"Local Storage","source":"277355b5a1564354.json","type":"application/json","size":548},{"uid":"899f233794963d7a","name":"Full Page Screenshot","source":"899f233794963d7a.png","type":"image/png","size":1821648},{"uid":"386121e1f62c5b4e","name":"URL","source":"386121e1f62c5b4e.uri","type":"text/uri-list","size":23},{"uid":"2fea5579ae9aa0f8","name":"Network Logs","source":"2fea5579ae9aa0f8.json","type":"application/json","size":7932452}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":7,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1752285805460,"stop":1752285805460,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Workspaces"},{"name":"severity","value":"normal"},{"name":"story","value":"WorkSpaces Creation and Editing Functionality"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"workspaces_test"},{"name":"subSuite","value":"TestWorkspaces"},{"name":"host","value":"fv-az2246-978"},{"name":"thread","value":"2092-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.workspaces_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":12,"skipped":0,"passed":0,"unknown":0,"total":12},"items":[{"uid":"b5820559a4d23fba","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/711//#testresult/b5820559a4d23fba","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1752199292441,"stop":1752199292441,"duration":0}},{"uid":"a0a6e59463e82a07","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/710//#testresult/a0a6e59463e82a07","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1752112738203,"stop":1752112738203,"duration":0}},{"uid":"414b64d5f8661ba6","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/709//#testresult/414b64d5f8661ba6","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1752026333320,"stop":1752026333320,"duration":0}},{"uid":"f89369fd474f1867","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/708//#testresult/f89369fd474f1867","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751939866010,"stop":1751939866010,"duration":0}},{"uid":"6d31790e622916a5","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/707//#testresult/6d31790e622916a5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751853756528,"stop":1751853756528,"duration":0}},{"uid":"91614ddd91352789","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/706//#testresult/91614ddd91352789","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751767538543,"stop":1751767538543,"duration":0}},{"uid":"70c849852f97f361","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/705//#testresult/70c849852f97f361","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751680344882,"stop":1751680344882,"duration":0}},{"uid":"b8b51dca39d58b98","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/704//#testresult/b8b51dca39d58b98","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751594180704,"stop":1751594180704,"duration":0}},{"uid":"47cca6eae0948e99","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/703//#testresult/47cca6eae0948e99","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751507812602,"stop":1751507812602,"duration":0}},{"uid":"f9dfe6853a9e961b","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/702//#testresult/f9dfe6853a9e961b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751421392933,"stop":1751421392933,"duration":0}},{"uid":"67f66c8d408e6c81","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/701//#testresult/67f66c8d408e6c81","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751335676906,"stop":1751335676906,"duration":0}}]},"tags":[]},"source":"d01668d44578cbf4.json","parameterValues":[]}